test/blackbox-tests/test-cases/pkg/opam-var/opam-var-pkg.t

```diff
@@ -103,10 +103,10 @@ corresponding Dune version.
     (run echo 32 %{pkg:foo:dev})
     (run echo 33 %{pkg-self:opamfile})
     (run echo 34 %{pkg:foo:opamfile})
-    (run echo 35 %{pkg-self:with-test})
```

Comment:
This actually might be more correct than the new version. It all depends on how we want to handle with-test.